router/core/src/test/scala/io/buoyant/router/RouterTest.scala

```diff
@@ -120,7 +120,6 @@ class RouterTest extends FunSuite {
         val id: Any = path match {
           case Path.Utf8("0") => Path.Utf8("some", "path")
           case Path.Utf8("1") => "string"
-          case Path.Utf8("2") => null
```

Comment:
finagle now calls `.hashCode` on ids which means that using `null` for an id is a no-no.